packages/kit/src/runtime/app/env.js

```diff
@@ -1,28 +1,11 @@
-/* global __SVELTEKIT_BUILD__ */
-
 /**
  * @type {import('$app/env').browser}
  */
 export const browser = !import.meta.env.SSR;
 
-/**
- * @type {import('$app/env').server}
- */
-export const server = !!import.meta.env.SSR;
-
 /**
  * @type {import('$app/env').dev}
  */
-export const dev = !__SVELTEKIT_BUILD__;
-
-/**
- * @type {import('$app/env').prod}
- */
-export const prod = __SVELTEKIT_BUILD__;
-
-/**
- * @type {import('$app/env').mode}
- */
-export const mode = import.meta.env.MODE;
+export const dev = __SVELTEKIT_DEV__;
```

Comment:
It corresponds to `vite dev`, which is the only usage that most people care about. `command` is only relevant to Vite plugin authors. I agree that `serve` vs `build` is a confusing distinction when the CLI commands are `dev` and `build`, but it's better to contain that confusion to Vite plugins than spread it into SvelteKit, I think

Reply:
What if we use `build`? That would be consistent with either viewpoint which would reduce the chances people get confused

Reply:
> > what's the difference to `browser` which also is just an alias to `import.meta.env.X`?
> 
> In terms of the underlying mechanism, `browser` _is_ just derived from `import.meta.env.SSR`, but I think it's categorically different to `mode`. For me, the litmus test is this: if we were to enable SvelteKit to work with other build tools, would it continue to make sense? `browser` vs `!browser` is fundamental — you'd expect that to do the same thing regardless of tooling. But `mode` is only meaningful in the context of Vite. If someone took an existing SvelteKit app and started using a hypothetical alternative build tool, they wouldn't be surprised that `import.meta.env.MODE` no longer worked, but they _would_ be surprised if `mode` no longer did anything useful.
> 
> That's not to say I'm proposing alternative build tools, it's really more of a thought experiment to figure out what falls into SvelteKit's remit and what doesn't.

It feels more correct not having `mode` as a part of `$app/env` now that the Vite config is separated from the sveltekit config. I also agree that the `server` and `prod` env-variables should be removed in favour of having a less confusing API, even though i still think that `if(server)` is much cleaner than `if(!browser)`. But implementing this in user-land is a piece of cake.

Reply:
It would be unusual. This is very common:

```js
if (dev) {
  do_some_extra_logging_or_something();
}
```

This is basically unheard-of:

```js
if (build) {
  do_some_extra_logging_or_something();
}
```

Sure, you _could_ do `!build` but I think it makes more sense (and is more aesthetically appealing) to default to the more common positive condition, which is both `browser` and `dev`

Reply:
fair enough. I suggested some extra docs around this, which might be enough, so will resolve this comment